src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -129,25 +127,29 @@ public HikariPool(final HikariConfig config)
 
       LinkedBlockingQueue<Runnable> addQueue = new LinkedBlockingQueue<>(config.getMaximumPoolSize());
       this.addConnectionQueue = unmodifiableCollection(addQueue);
-      this.addConnectionExecutor = createThreadPoolExecutor(addQueue, poolName + " connection adder", threadFactory, new ThreadPoolExecutor.DiscardPolicy());
-      this.closeConnectionExecutor = createThreadPoolExecutor(config.getMaximumPoolSize(), poolName + " connection closer", threadFactory, new ThreadPoolExecutor.CallerRunsPolicy());
+      this.addConnectionExecutor = createAddConnectionExecutor(threadFactory, addQueue);
+      this.closeConnectionExecutor = createThreadPoolExecutor(config.getMaximumPoolSize(), poolName + " connection " +
+            "closer",
+         threadFactory, new ThreadPoolExecutor.CallerRunsPolicy());
 
       this.leakTaskFactory = new ProxyLeakTaskFactory(config.getLeakDetectionThreshold(), houseKeepingExecutorService);
 
-      this.houseKeeperTask = houseKeepingExecutorService.scheduleWithFixedDelay(new HouseKeeper(), 100L, housekeepingPeriodMs, MILLISECONDS);
-
-      if (Boolean.getBoolean("com.zaxxer.hikari.blockUntilFilled") && config.getInitializationFailTimeout() > 1) {
-         addConnectionExecutor.setCorePoolSize(Runtime.getRuntime().availableProcessors());
-         addConnectionExecutor.setMaximumPoolSize(Runtime.getRuntime().availableProcessors());
+      this.houseKeeperTask = houseKeepingExecutorService.scheduleWithFixedDelay(new HouseKeeper(), 100L,
+         housekeepingPeriodMs, MILLISECONDS);
+   }
 
-         final long startTime = currentTime();
-         while (elapsedMillis(startTime) < config.getInitializationFailTimeout() && getTotalConnections() < config.getMinimumIdle()) {
-            quietlySleep(MILLISECONDS.toMillis(100));
-         }
+   private ThreadPoolExecutor createAddConnectionExecutor(ThreadFactory threadFactory,
+                                                          LinkedBlockingQueue<Runnable> addQueue)
```

Comment:
Do not wrap line here.  We allow lines of up to 220 characters.

Parameters should be ``final``.